### DISCUSSION TITLE
Add ability to switch between computer screen

### DISCUSSION THREAD

**Discussion by Biokax** [ppy/osu#14933] [Ideas] [answered]
Hey,

For no reason my osu lazer openned on my 2nd screen and I don't know how to switch to the main screen because it don't have the option to choose the playable screen. 
QOL content I know but does it have a way to fix it ?

**Comment by bdach** [comment, marked as answer]
try switching to windowed mode, dragging the window to the other screen and going to full screen again.

**Reply by Biokax** [reply]
that works, thanks. Do you know if a screen selector will come one day ?

**Reply by bdach** [reply]
the framework already supports doing this, it just isn't exposed as a setting yet. there's no reason why it wouldn't be there, i think (but it would probably only be present in fullscreen/borderless).
